jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Path mapping can affect editor integration. For example, VS Code uses the TypeScript language service to provide productivity tooling like go-to-definition, auto-import of modules, and so on. This new file is necessary for these features to continue working. It's a popular editor these days so it might be worth baking support into the repo, but understandable if you prefer otherwise!

Reply:
I use VSCode myself so I'm perfectly all right with baking that support in. Is that the approach you're taking here or would that be a further update?

Reply:
Yep, that's the approach in place here. No further updates involved